### ISSUE TITLE
Support paths for SSL cert/key

### ISSUE THREAD

**Issue by zumm** [nuxt/framework#3949]
At this moment nuxt supports cert/key only as content (NITRO_SSL_CERT/NITRO_SSL_KEY env vars). Can support of paths be added too? It would improve DX with using .env file.

**Comment by danielroe** [comment]
This would increase the size of the default Nitro server, and it would have to include FS access, which wouldn't work in non-Node environments. But you can easily implement this yourself.

```sh
NITRO_SSL_CERT=$(cat cert.crt) NITRO_SSL_KEY=$(cat key.key) node .output/server/index.mjs
```

**Comment by zumm** [comment]
Yes, but it requires extract running command from scripts section of package.json or make it cross-platform in some way.

Can dotenv dependency be updated to v15 for multilane value support at least?

**Comment by pi0** [comment]
`.env` file is not loaded with the production bundle of nitro. 

You can create a `server.mjs` file for cross platform support:

```js
#!/bin/env/node
import fs from 'fs'

process.env.NITRO_SSL_CERT = fs.readFileSync('server.crt')
process.env.NITRO_SSL_KEY = fs.readFileSync('server.key')

import('./.output/server/index.mjs').catch(err => {
   console.error(err)
   process.exit(1)
})
```
